charter.html

```diff
@@ -0,0 +1,290 @@
+    </h1>
+    <p>
+      This Charter is work in progress. To submit feedback, please use <a
+      href="https://github.com/w3c/ColorWeb-CG/issues">https://github.com/w3c/ColorWeb-CG/issues</a>
+      where the Charter is being developed.
+    </p>
+    <ul>
+      <li>This Charter: <i>n/a</i>
+      </li>
+      <li>Previous Charter: <i>n/a</i>
+      </li>
+      <li>Start Date:  <time datetime="2017-01-27">Fri, 27 Jan 2017</time></span>
+      </li>
+      <li>Last Modified: <i>n/a</i></span>
+      </li>
+    </ul>
+    <h2 id="goals">
+      Goals
+    </h2>
+    <p>
+      The goal is to allow color experts from various fields - including
+      print-oriented color management, display, media and entertainment
+      community, CSS , image and video coding and graphics experts - to come
+      together, share ideas, use cases, experiences and discuss technical
+      solutions to improve the state of color on the Web.
+    </p>
+    <h2 id="scope-of-work">
+      Scope of Work
+    </h2>
+    <p>The works addresses the representation and processing of color
+    information as applied to the Web.</p>
+    <p>
+      Particular areas of interest include, but are not limited to:
+      <ul>
+        <li>Unambiguous, colorimetric color specification of desired presentation</li>
+        <li>Wide gamut colorspaces (wider than sRGB)</li>
+        <li>High dynamic range (greater range of luminance, darker blacks and
+        peak whites greater than paper white)</li>
+        <li>Color matching between media (raster images, video, vector graphics)
+        and text/styling content (styled by CSS)</li>
+        <li>Consistency and predictability of display</li>
+      </ul>
+    </p>
+    <h3 id="out-of-scope">
+      Out of Scope
+    </h3>
+    <p><i>n/a</i></p>
+    <h2 id="deliverables">
+      Deliverables
+    </h2>
+    <h3 id="specifications">
+      Specifications
+    </h3>
+    <ul>
+      <li>Extensions to Canvas/WebGPU/WebGL APIs for HDR support</li>
+    </ul>
+    <h3 id="non-normative-reports">
+      Non-Normative Reports
+    </h3>
+    <ul>
+      <li>CG report</li>
+      <li>Requirements for adding HDR support in PNG</li>
+      <li>Explainer for adding floating point support to HTML canvas</li>
+    </ul>
+    <h3 id="test-suites">
+      Test Suites and Other Software
+    </h3>
+    <p>
+      The group MAY produce test suites to support the Specifications. Please
+      see the GitHub LICENSE file for test suite contribution licensing
+      information.
+    </p>
+    <h2 id="liaisons">
+      Dependencies or Liaisons
+    </h2>
+    <p>
+      The group expects to liaise with the following groups:
+      <ul>
+        <li><a href="https://www.w3.org/groups/wg/htmlwg">W3C HTML WG</a></li>
+        <li><a href="https://www.w3.org/groups/wg/png">W3C PNG WG</a></li>
+        <li><a href="https://www.w3.org/2011/webtv/wiki/Main_Page">W3C Media and Entertainment Interest Group</a></li>
```

Comment:
This one should be the same format as the other links to W3C groups:

https://www.w3.org/groups/ig/me